src/components/IFrame.js

```diff
@@ -100,3 +124,12 @@ export default function ReportScreen() {
         />
     );
 }
+
+OldDotIFrame.propTypes = propTypes;
+
+export default withOnyx({
+    session: {
+        key: ONYXKEYS.SESSION,
+        selector: (session) => session,
```

Comment:
Any reason for using `selector` here?

Reply:
no reason at all. forgot to fix that 😆

Reply:
https://github.com/Expensify/App/pull/29089/commits/a0e3fe1a35aa8e4d452271b55af7a847b66bcf47